Makefile

```diff
@@ -1,3 +1,3 @@
 compile:
 	mkdir -p target
-	cp sbtest.sh target/sbtest.sh
+	src/aggregate.sh > target/sbtest.sh
```

Comment:
Ah got it... you need to make aggregate executable. Also, I would put it out of src/